src/OutboundServer/api.yaml

```diff
@@ -623,7 +629,7 @@ components:
           type: string
           description: Transaction ID from the DFSP backend, used to reconcile transactions between the Switch and DFSP backend systems.
         from:
-          $ref: '#/components/schemas/transferParty'
+          $ref: '#/components/schemas/transferPartyFrom'
         to:
           $ref: '#/components/schemas/transferParty'
```

Comment:
the "to" transfer party will become populated from the PUT /parties response and returned to the caller, I would suggest using a similar pattern to transferPartyFrom e.g. transferPartyTo so the DFSP making the outbound request will see the extensions returned from the payee DFSP.

Reply:
Changes made as suggested.